### PR TITLE
Update the register cluster section to provide more clarity

### PR DESCRIPTION
**Pull request by egrosdou01**
Update the register cluster section to provide more clarity to the end user on the two ways of registering a managed cluster with Sveltos.

### CODE REVIEW

File: docs/register/register-cluster.md [hunk 1]
```diff
@@ -10,56 +10,55 @@ tags:
...
 ```
-sveltosctl register cluster --namespace=<namespace> --cluster=<cluster name> --fleet-cluster-context=<context name> --labels=key1=value1,key2=value2
+$ sveltosctl register cluster --namespace=<namespace> --cluster=<cluster name> --fleet-cluster-context=<context name> --labels=key1=value1,key2=value2
 ```
 
-This single command will generate a kubeconfig file and register your cluster with Sveltos.
+The above command will generate a kubeconfig file and register the cluster with Sveltos.
 
-Otherwise follow those steps:
+If the kubeconfig of the managed cluster is not available, use the below steps to generate a new one with the right permissions and perform a registration with Sveltos.
```
Comment (gianlucam76) [inline docs/register/register-cluster.md:29]: I am confused about this "If the kubeconfig of the managed cluster is not available" cause we need a kubeconfig to access the managed cluster and run command 1

**Review by gianlucam76 — COMMENTED** [review]
Thank you. Looks great. I only have one single comment (likely I misunderstood)

File: docs/register/register-cluster.md (hunk 1 above)
Comment (egrosdou01) [reply docs/register/register-cluster.md:29]: My understanding is that the additional commands will generate the correct kubeconfig (new managed cluster) and then use it to register the cluster with Sveltos. If my understanding is not correct, I will have to rephrase it.

**Comment by egrosdou01** [comment]
Understood, let me rephrase the text and push the changes.

File: docs/register/register-cluster.md (hunk 1 above)
Comment (egrosdou01) [reply docs/register/register-cluster.md:29]: Understood, let me rephrase the text and push the changes.

Comment (gianlucam76) [reply docs/register/register-cluster.md]: thank you!

**Review by gianlucam76 — APPROVED** [review]
Thank you!
